src/main/java/com/moabam/api/application/room/CertificationService.java

```diff
@@ -169,4 +167,14 @@ private int getRoomLevelAfterExpApply(int roomLevel, Room room) {
 
 		return room.getLevel();
 	}
+
+	private void provideBugToCompletedMembers(BugType bugType, List<DailyMemberCertification> dailyMemberCertifications,
+		int expAppliedRoomLevel) {
+		List<Long> memberIds = dailyMemberCertifications.stream()
+			.map(DailyMemberCertification::getMemberId)
+			.toList();
```

Comment:
```suggestion
		List<Long> memberIds = map(dailyMemberCertifications, DailyMemberCertification::getMemberId);
```
C: StreamUtils 만들어놓은 것 적용할 수 있을듯!

Reply:
XX : 난 안쓰는게 더 조아!

(제 말 아님)